backend/src/main/java/gov/cdc/usds/simplereport/service/PatientLinkService.java

```diff
@@ -33,21 +36,47 @@ public PatientLink getPatientLink(UUID internalId) {
             () -> new IllegalGraphqlArgumentException("No patient link with that ID was found"));
   }
 
+  public PatientLink getRefreshedPatientLink(UUID internalId) {
```

Comment:
Given that this is the mechanism that a provider would use to "re-send" a link, should we also have this reset the failed attempts?

Reply:
Yeah, I think that's a good idea! I can add :+1: